.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "Ruby",
+	"build": {
+		"dockerfile": "base.Dockerfile",
+		"args": {
+			"VARIANT": "3-buster"
+		}
+	},
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": {},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"rebornix.Ruby"
```

Comment:
Should we add https://marketplace.visualstudio.com/items?itemName=sorbet.sorbet-vscode-extension?